File: vector/tempCodeRunnerFile.dart
#include<iostream>
#include<vector>
using namespace std;
void add(string num1,string num2){
    int n1=num1.size();
    int n2=num2.size();
    if(n1>n2){
        swap(num1,num2);
        swap(n1,n2);
    }
    string res="";
    int carry=0;
    for(int i=0;i<n1;i++){
        int sum=(num1[i]-'0')+(num2[i]-'0')+carry;
        res.push_back(sum%10+'0');
        carry=sum/10;
    }
    for(int i=n1;i<n2;i++){
        int sum=(num2[i]-'0')+carry;
        res.push_back(sum%10+'0');
        carry=sum/10;
    }
    if(carry){
        res.push_back(carry+'0');
    }
    cout<<res<<endl;
}



int main(){
     string num1="123",num2="11";
      add(num1,num2);

    return 0;
}

File: vector/tempCodeRunnerFile.cpp
#include<iostream>
#include<algorithm>
#include<vector>
using namespace std;
  int median(vector<int>& num1 , vector<int>& num2){
    int n=num1.size();
    int m=num2.size();
    vector<int>num3;
    // copy num1 to num3
    for(int i=0;i<n;i++){
        num3.push_back(num1[i]);
    }
    // copy num2 to num3
    for(int i=0;i<m;i++){
        num3.push_back(num2[i]);
    }
    // sort the num3 array
    sort(num3.begin(),num3.end());

    int total =num3.size();
    if(total%2==0){
        int mid1=total/2;
        int mid2=mid1-1;
        return (num3[mid1]+num3[mid2])/2.0;
    }else{
        int mid=total/2;
        return num3[mid];
    }
  }
int main(){
    vector<int>num1={1,2,3};
    vector<int>num2={6,8};
    cout<<median(num1,num2)<<endl;
    return 0;
}
